### ISSUE TITLE
[PWA] Checkout

### ISSUE THREAD

**Issue by herzog31** [hlxsites/maidenform#188]
- [x] Dedicated checkout page
- [ ] Custom template with no navigation in header (checkout funnel)
- [ ] Loqate to autofill addresses
- [ ] Hardcode links to list of terms & condition pages. No checkbox required.
- [x] Magento Storefront Events (see #90) 
- [ ] Cancel order button once API is ready from Maidenform team, see #210 

![SCR-20230412-d85](https://user-images.githubusercontent.com/4084418/231384918-6202ffb1-0c72-4d13-8805-d5e9726906b0.png)

Out of scope: #189
